fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,13 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+extern crate bytebeat;
+
+fuzz_target!(|data: &[u8]| {
+    if let Ok(s) = std::str::from_utf8(data) {
+        if let Ok(program) = bytebeat::parse_beat(s) {
+            if let Ok(program) = bytebeat::compile(program) {
+                let _ = bytebeat::eval_beat(&program, 0.0);
```

Comment:
We want to evaluate our program over different data. This current setup won't catch any bugs that depend on the value of `t`, so we need the fuzzer to be able to control `t`. We could probably get this by splitting off the first value of the data as a number.